.github/workflows/ci.yml

```diff
@@ -14,11 +14,14 @@ on:
 name: CI
 
 jobs:
-  docker_smoketests:
-    name: Smoketests
+  find_ref:
+    name: Find Git ref
```

Comment:
was there a reason we separated this out? I like having stuff factored, but it adds an extra line in the list of checks in the PR review, which makes me a bit sad:
![image](https://github.com/user-attachments/assets/2de79c1e-7dd0-470b-8d42-c92a64075daa)

Reply:
I wasn't sure how to port it to Powershell, so I did this. I might be able to make it run in sh even on windows though

Reply:
ahh it runs on a different runner. that makes sense. eh, probably fine for now then.

maybe there's a less dumb way to get the git ref.. I'm pretty sure I added this.

Reply:
honestly.. I'm not even 100% sure why we need to get the ref at all.. at first glance it looks like we're recreating the default behavior.

anyway probably fine to leave it for now.

Reply:
Oh it's so we can trigger it manually from the `Actions` page